java/vector/src/main/java/org/apache/arrow/vector/VectorSchemaRoot.java

```diff
@@ -72,6 +92,9 @@ public VectorSchemaRoot(Schema schema, List<FieldVector> fieldVectors, int rowCo
     }
   }
 
+  /**
+   * Creates a new set of vectors corresponding to schema that are empty.
```

Comment:
maybe this would sound better "Creates a new set of empty vectors corresponding to the given schema"?

Reply:
Seems reasonable. changed (will be in the next push as with addressing your other comments.).